### ISSUE TITLE
Use easy-to-filter Log tag

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#295]
Log excerpt:
```
10-08 15:46:32.312 15692 16024 E Volley  :      at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:123)
10-08 15:46:32.313 15692 15692 E fr.free.nrw.commons.upload.MwVolleyApi$LogResponseErrorListener: com.android.volley.VolleyError: java.lang.RuntimeException: No-args constructor for class fr.free.nrw.commons.upload.MwVolleyApi$Page does not exist. Register an InstanceCreator with Gson for this type to fix this problem.
10-08 15:46:37.525 15692 15692 D fr.free.nrw.commons.upload.ShareActivity: Filepath: /storage/emulated/0/DCIM/OpenCamera/IMG_20161004_141951.jpg
10-08 15:46:37.525 15692 15692 D fr.free.nrw.commons.upload.ShareActivity: Calling GPSExtractor
10-08 15:46:37.527 15692 15692 D fr.free.nrw.commons.upload.GPSExtractor: EXIF data has location info
10-08 15:46:37.527 15692 15692 D Image   : Latitude: 35/1,39/1,39/1 N
10-08 15:46:37.527 15692 15692 D Image   : Longitude: 139/1,43/1,36/1 E
10-08 15:46:37.527 15692 15692 D Coords  : Latitude and Longitude are 35.66083333333333|139.72666666666666
```
Filtering via keywords is difficult because of the necessity to catch various keywords such as "Volley", "commons", "Image", "Coords".
How about using the same prefix for all of our tags?

**Comment by misaochan** [comment]
I agree, I should have done that from the start. Will change whatever I can find.

**Comment by misaochan** [comment]
Should be fixed in #296, let me know if I missed anything.
